terraform/.terraform-version

```diff
@@ -1 +1,2 @@
-1.5.5
+
+1.5.5
```

Comment:
Please set your editor to end each file with a newline. There should be no changes to this .terraform-version file in this PR.

Reply:
> Please set your editor to end each file with a newline. There should be no changes to this .terraform-version file in this PR.

Resolved